forward_engineering/utils/general.js

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
we already have `forward_engineering/helpers/utils.js`, could it be placed at least under the "helpers" folder, and have more explicit name, something like `statementsJoiner.js`?

Reply:
We started implementing a new plugin structure. There must be only 1 utility file with functions that used by FE and Alter script, and it is `general/utils.js`

Reply:
> We started implementing a new plugin structure
Where can I check the agreed approach with that 1 file limitation?)

Reply:
https://app.gitbook.com/o/HBtg1gLTy0nw4NaX0MaV/s/RsuO9R6FbfIJtofdqr4L/~/changes/213/guides-and-how-to/plugins/alter-and-fe-script.-directory-structure-and-general-flow